protocol/rpcconsumer/relay_processor.go

```diff
@@ -18,16 +18,17 @@ import (
 	spectypes "github.com/lavanet/lava/v2/x/spec/types"
 )
 
-const (
-	MaxCallsPerRelay                   = 50
-	NumberOfRetriesAllowedOnNodeErrors = 2 // we will try maximum additional 2 relays on node errors
-)
-
 type Selection int
 
 const (
-	Quorum     Selection = iota // get the majority out of requiredSuccesses
-	BestResult                  // get the best result, even if it means waiting
+	Quorum           Selection = iota // get the majority out of requiredSuccesses
+	BestResult                        // get the best result, even if it means waiting
+	MaxCallsPerRelay = 50
```

Comment:
you've put it inside the Selection enum, it was intentional the const was in a different place